stable/insights-agent/templates/aws-costs/cronjob.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.awscosts.enabled -}}
+{{- $_ := set . "Label" "awscosts" }}
+{{- $_ := set . "Config" .Values.awscosts }}
+
+{{ if .Values.awscosts.awsAccessKeyId }}
+apiVersion: v1
+kind: Secret
```

Comment:
we should break this out into a `secret.yaml`

Reply:
done